deploy/kubernetes/filebeat-kubernetes.yaml

```diff
@@ -123,15 +123,19 @@ data:
                 logs_path: "/var/log/containers/"
 
     # To enable hints based autodiscover, remove `filebeat.inputs` configuration and uncomment this:
-    #filebeat.autodiscover:
+    # filebeat.autodiscover:
```

Comment:
is this extra space intentional?

Reply:
By default, when I commented in and out the autodiscover block, it added this space. TBH it looks more readable